test/load/packer/consul-ami/README.md

```diff
@@ -0,0 +1,16 @@
+# Consul AMI
+
+## Quick start
+
+To build the Consul AMI:
+
+1. `git clone` this repo to your computer.
+2. Install [Packer](https://www.packer.io/).
+3. Configure your AWS credentials using one of the [options supported by the AWS
+   SDK](http://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/credentials.html). Usually, the easiest option is to
+   set the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment variables.
```

Comment:
I would also explicitly set and read `AWS_REGION` from the environment in the packer files since those are hard coded to us-east-2 and if it differs, users will have to change it in 2 places.